### ISSUE TITLE
no python 3.10 on M1 Macs

### ISSUE THREAD

**Issue by conversy** [christofmuc/KnobKraft-orm#182]
Hi,

The brew home directory has changed on M1 mac, it's not ` /usr/local/opt` anymore, but rather `/opt/homebrew`.

here:
https://github.com/christofmuc/KnobKraft-orm/blob/20aea14236dca045060658023036e6392f49d90c/adaptions/GenericAdaptation.cpp#L234

`/usr/local/opt/python3/Frameworks/Python.framework/Versions/3.10`

should be tested but if it fails, another test should be performed with something like: 

`/opt/homebrew/Cellar/python\@3.10/3.10.7/`

Note:
```
$ ls /opt/homebrew/Cellar/python\@3.10/3.10.7/
Frameworks/            INSTALL_RECEIPT.json   Python Launcher 3.app/ bin/                   libexec/
IDLE 3.app/            LICENSE                README.rst             lib/                   share/
$ ls /opt/homebrew/Cellar/python\@3.10/3.10.7/Frameworks/Python.framework/Versions/3.10
Headers@        Python*         Resources/      _CodeSignature/ bin/            include/        lib/            share/
```

Another suggestion: you might use the result of the execution of `python3 -c "import sys; print(sys.executable)"` which gives on my machine `/opt/homebrew/opt/python@3.10/bin/python3.10`.

I do not know what is recommended though to correctly find a suitable python installation and embed it in.

Best,

Stéphane

**Comment by christofmuc** [comment]
@conversy Thank you, yes, the Python on Mac issue is a long standing cause of sadness. We have a PR#187 now which implements a list of search paths, which I think wll be a good solution until I figured it out how to embed Mac with the App like I do it on Windows!

**Comment by christofmuc** [comment]
Released new search path with the 2.0.0 release. Also note there is the ORM_PYTHON environment variable that can be set so the Python can be located anywhere on your Mac.
